src/Applications/LDAS_App/ldas_setup

```diff
@@ -1438,30 +1446,42 @@ def _printExeInputKeys(rqdExeInpKeys):
     print '# Coupling of LDAS to ADAS ("LADAS"):                      #'
     print '#                                                          #'
     print '#   0 -- LDAS not coupled with ADAS (default)              #'
-    print '#   1 -- LDAS coupled with ADAS ${ADAS_exp}                #'
-    print '#                                                          #'
-    print '#  Requirements for LADAS_COUPLING = 1:                    #'
+    print '#   1 -- LDAS coupled with central member of ADAS          #'
+    print '#   2 -- LDAS coupled with ens component of ADAS           #' 
+    print '#                                                          #' 
+    print '#  Requirements for LADAS_COUPLING > 0:                    #'
+    print '#                                                          #'     
+    print '#  (0) Specify ADAS_EXPDIR = [full_path]/[ADAS_EXPID]      #'
     print '#                                                          #'
-    print '#  (1) BEG_DATE must be consistent with ${ADAS_exp}        #'
-    print '#                first cycle date(21z, 3z, 9z or 15z)      #'
+    print '#  (1) BEG_DATE must be consistent with first cycle date   #'
+    print '#        and time of ADAS experiment (time is typically    #'
+    print '#        3z, 9z, 15z, or 21z)                              #'
     print '#                                                          #'        
-    print '#  (2) EXP_DOMAIN must be global CS grid as in ${ADAS_exp} #'
+    print '#  (2) EXP_DOMAIN must be global CS grid as in ADAS exp    #'
     print '#                                                          #'
-    print '#  (3) MET_TAG  must be set to  ${ADAS_exp}__Nx+-          #'
-    print '#      MET_PATH must be set to  ${ADAS_exp}/lana/forc      #' 
-    print '#    After ldas exp setup, verify the following link:      #'        
-    print '#    ../input/met_forcing/forc -> ${ADAS_exp}/lana/forc    #'
+    print '#  (3) MET_TAG  must be set to [ADAS_EXPID]__Nx+-          #'
+    print '#      MET_PATH must be set as follows for                 #'
+    print '#        LADAS_COUPLING = 1:                               #'        
+    print '#          [ADAS_EXPDIR]/lana/forc                         #'        
```

Comment:
Are the "./lana/forc" and "./atmens/ensdiag/forc" sub-directories hardwired into the ADAS? 
The "./atmens/ensdiag/forc" sub-directory seems different from what used to be hard-coded in "average_ensemble_forcing.py".  Is the dir name actually correct here?

Reply:
@gmao-rreichle   ./lana/ forc  is hardwired in ADAS central, it links to fvwork when the run is going.  ./atmens/ensdiag/forc  is not hardwired in ADAS, it gets created by the python code to hold the output of py averaging, so technically it does not have to be within ADAS dir.   An advantage of this specific path dir is  it is put within atmens existing stage dir structure, it stays there and gets archived along with atmens.

Reply:
@saraqzhang : Thanks for the info.  One more question going deeper into details... Is the ./lana/forc directory hardwired into _your_ version of the ADAS, or is it already in an official ADAS tag?  Perhaps this somehow made it into an official ADAS tag when Clara was working on the LADAS, in which case we should just work with it as is.  If the ./lana/forc directory is not already in an official ADAS tag, it may not survive the pre-merger code review, but we can deal with it then.  Either way, for now it's fine as is.

Reply:
@gmao-rreichle :  ./lana was in the  original ADAS tag ( in fvsetup,  make the dir).    ./lana/forc is created in the process of running g5adas  ( point to fvwork/ ), and removed when the run finishes ( fvwork is gone by then).